python/pyproject.toml

```diff
@@ -29,16 +29,16 @@ coloredlogs = "^15.0"
 pandas = "^1.5"
 pyarrow = "^10.0"
 # fabric
-misp-stix = {version = "^2.4", optional = true}
+misp-stix = {version = "^2.4"}
```

Comment:
did you identify why this optional dependency was leaking into the core? It would really be better if we could keep this optional

Reply:
If this is optional, `stix2` is missing when doing `import vast`.

We could probably have an explicit dependency for stix2 instead, let me try that.

Reply:
that seems to work, resolving. You have time until the build passes to raise additional concerns ;-)